packages/subgraph/schema.graphql

```diff
@@ -42,21 +42,14 @@ type DayStatSnapshot @entity {
   totalCollateralAmount: BigInt!
 }
 
-enum Action {
-  WITHDRAW_COLLAT
-  DEPOSIT_COLLAT
-  MINT_SHORT
-  BURN_SHORT
-  LIQUIDATE
-}
-
 type VaultHistory @entity {
   id: ID!
+  txid: String!
   timestamp: BigInt!
   totalEthCollateralAmount: BigInt!
   oSqthAmount: BigInt!
   ethCollateralAmount: BigInt!
-  action: Action!
+  action: String!
```

Comment:
Not using enums anymore?

Reply:
I experienced some compiling errors with enum Action, I can check that again